### PR TITLE
Feature/plant detail engine

### PR DESCRIPTION
**Pull request by SajuJ**
Created foundation for plant workflow engine, added INNS workflow strategy and Prohibited workflow strategy.

### CODE REVIEW

**Review by asakthivel1810 — APPROVED** [review]

File: src/helpers/db/update-db-plant.js
Comment (manabshy) [inline src/helpers/db/update-db-plant.js:1]: You're overwriting this change ?. Please merge with the latest main branch.

File: src/helpers/db/create-ds-indexes.js
Comment (manabshy) [inline src/helpers/db/create-ds-indexes.js:1]: Same - Please revert this change. You're overwriting this change

**Review by manabshy — CHANGES_REQUESTED** [review]

File: src/helpers/mongodb.js
Comment (manabshy) [inline src/helpers/mongodb.js:1]: This is not required. Please take the changes from main

**Review by manabshy — CHANGES_REQUESTED** [review]
